ssas/service/main/main.go

```diff
@@ -78,8 +81,15 @@ func init() {
 	const usageListExpCreds = "list credentials about to expire or timeout due to inactivity"
 	flag.BoolVar(&doListExpCreds, "list-exp-creds", false, usageListExpCreds)
 
+	const usageShowXData = "display ACOs group xdata"
```

Comment:
To eliminate usage of ACO, maybe change to `display group xdata` to be more general.